app/src/main/java/com/example/vmedvediev/ua21summerdancecamp/model/LocalStorage.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.vmedvediev.ua21summerdancecamp.model
+
+object LocalStorage {
+
+    val eventsList = ArrayList<ListItem>()
+
+    fun getAmountOfEventsByDate(date: String) : Int {
+        var amountOfEvents = 0
+        eventsList.forEach {
+            if (it.getDateOfEvent() == date) {
+                amountOfEvents++
+            }
+        }
+        //We increment result in order to add date object.
+        return ++amountOfEvents
+    }
+
+    fun getEventsByDate(date: String) : ArrayList<ListItem> {
+        val sortedByDateList = ArrayList<ListItem>()
```

Comment:
Wrong approach
1) to sort list by date you should use interface comparator and define sorting rule.
Then you can use just Collections.sort(array)
2) Kotlin has built-in functions for sorting (ascending or descending)